ApplicationLibCode/FileInterface/RifEclipseSummaryAddress.cpp

```diff
@@ -49,28 +50,34 @@ RifEclipseSummaryAddress::RifEclipseSummaryAddress( SummaryVarCategory
     {
         case SUMMARY_REGION:
             m_regionNumber = RiaStdStringTools::toInt16( identifiers[INPUT_REGION_NUMBER] );
+            m_id           = RiaStdStringTools::toInt( identifiers[INPUT_ID] );
```

Comment:
Assignment of m_id is valid for all variants, consider moving it outside the switch